code/game/objects/structures/morgue.dm

```diff
@@ -264,9 +264,9 @@
 	else if (src.locked == 0)
 		playsound(src.loc, 'sound/items/Deconstruct.ogg', 50, 1)
 		src.connected = new /obj/structure/c_tray( src.loc )
-		step(src.connected, SOUTH)
+		step(src.connected, dir)
 		src.connected.layer = OBJ_LAYER
-		var/turf/T = get_step(src, SOUTH)
+		var/turf/T = get_step(src, dir)

```

Comment:
Ah right i should do the // Edit thing. That fucked up the cremator so fucking badly. Hardcoding it in a way that the Cremator Tray would ALWAYS point south no mater the direction the crematorium was positioned.